apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-item.js

```diff
@@ -49,18 +56,67 @@ const TemplateSelectorItem = props => {
 
 	const labelId = `label-${ id }-${ value }`;
 
+	/**
+	 * Determines (based on whether the large preview is able to be visible at the
+	 * current breakpoint) whether or not the Template selection UI interaction model
+	 * should be select _and_ confirm or simply a single "tap to confirm".
+	 * The reason for this is that on larger screens the large preview is visible
+	 * which necessitates the double interaction. Without this then only a single tap
+	 * is necessary.
+	 *
+	 * @param  {string} tplSlug template slug string from the button value attr
+	 * @param  {string} tplName template name string from the button text
+	 */
+	const handleLabelClick = ( tplSlug, tplName ) => {
+		const largeTplPreviewVisible = window.matchMedia( '(min-width: 660px)' ).matches;
```

Comment:
I was concerned about this but it seems to be pretty solid: https://caniuse.com/#feat=matchmedia